tests/fast_api/test_fast_route.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+
+import aiohttp
+
+
+def test_read_main(client):
+    async def test_async():
+        async with aiohttp.ClientSession() as session:
+            async with session.get(
+                f"http://{client.service_client.config.target_host}/"
```

Comment:
I wonder if you want to run on another port that's not port 80 for your sample and for this test